devicetypes/smartthings/ezex-smart-occupancy-sensor.src/ezex-smart-occupancy-sensor.groovy

```diff
@@ -0,0 +1,106 @@
+/*
+ *  Copyright 2019 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ *  Author : Fen Mei / f.mei@samsung.com
+ *  Date : 2019-02-12
+ */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+metadata {
+    definition(name: "eZEX Smart Occupancy Sensor(AC Type)", namespace: "smartthings", author: "SmartThings", runLocally: false, mnmn: "SmartThings", vid: "SmartThings-smartthings-ZigBee_Dimmer_with_Motion_Sensor") {
```

Comment:
the name field must match the filename.

I'd prefer you used a more generic filename like "zigbee-plugin-motion-sensor" and just left your join name as is.

Why use this very long vid that has extra capabilities instead of using something like `generic-motion-2`?

Reply:
The device is AC type, will not report battery, but generic-motion-2 include battery remain perecentage